pkgs/test_core/lib/src/runner/vm/test_compiler.dart

```diff
@@ -152,13 +152,12 @@ class _TestCompilerForLanguageVersion {
   Future<CompileResult?> _createCompiler(Uri testUri) async {
     final platformDill = 'lib/_internal/vm_platform_strong.dill';
     final sdkRoot =
-        Directory(p.relative(p.join(Platform.resolvedExecutable, '..', '..')))
-            .uri;
+        p.relative(p.dirname(p.dirname(Platform.resolvedExecutable)));
```

Comment:
Why `p.relative` here?

Reply:
It likely isn't necessary, I was just retaining as much of the original code as possible to avoid changing the intent.

It might be removing drive letters or something in come cases? Idk if package:path would actually support that or not.